include/deal.II/fe/fe_base.h

```diff
@@ -592,7 +592,8 @@ class FiniteElementData
    */
   template <int structdim>
   unsigned int
-  n_dofs_per_object(const unsigned int i = 0) const;
+  n_dofs_per_object(
+    const unsigned int i = 0 /*numbers::invalid_unsigned_int*/) const;
```

Comment:
What is this about?

Reply:
This and the next was for "debugging". I have remove them.